wfe/web-front-end.go

```diff
@@ -708,7 +708,11 @@ func (wfe *WebFrontEndImpl) Certificate(response http.ResponseWriter, request *h
 
 		cert, err := wfe.SA.GetCertificateByShortSerial(serial)
 		if err != nil {
-			wfe.sendError(response, "Not found", err, http.StatusNotFound)
+			if strings.HasPrefix(err.Error(), "gorp: multiple rows returned") {
+				wfe.sendError(response, "Multiple certificates with same short serial", err, http.StatusInternalServerError)
```

Comment:
I think this is actually a positive thing, since we represent to our users through our CP that we will only ever issue two certificates with the same short serial. If we fail at that, we don't want to hide that fact from the world, but be upfront about the fact that we have an error and plan to fix it.